lib/_tls_common.js

```diff
@@ -143,7 +143,6 @@ exports.createSecureContext = function createSecureContext(options, context) {
   // Do not keep read/write buffers in free list
   if (options.singleUse) {
     c.singleUse = true;
-    c.context.setFreeListLength(0);
```

Comment:
This will have implications on a memory footprint of the `tls.connect()`. Since we use `SSL_MODE_RELEASE_BUFFERS`, the released buffers for such one-time contexts will be kept until the Garbage Collection.

I wonder if there is some way to turn off the freelist feature in OpenSSL 1.1.0?